app/src/main/java/com/github/se/orator/model/symblAi/SpeakingRepositoryRecord.kt

```diff
@@ -21,9 +21,14 @@ class SpeakingRepositoryRecord(private val context: Context) : SpeakingRepositor
   override val analysisState: StateFlow<SpeakingRepository.AnalysisState> = _analysisState
 
   // Functions to start and stop recording
-  override fun startRecording() {
+  override fun startRecording(audioFile: File) {
```

Comment:
Great idea! I haven't done this since I overrode an already existing method that didn't have this feature. Another PR to make the repository more secure should be done in the future.